hedera-mirror-rosetta/build/entrypoint.sh

```
@@ -19,6 +19,33 @@ function cleanup() {
   echo "Cleanup complete"
 }
 
+function init_db() {
+  echo "Initializing database"
+
+  PG_CLUSTER_CONF=/etc/postgresql/${PG_VERSION}/${PG_CLUSTER_NAME}
+  if [[ -f "${PGDATA}/PG_VERSION" ]]; then
+    # relink the config dir just in case it's a newly created container with existing mapped /data dir
+    ln -sTf ${PGDATA} ${PG_CLUSTER_CONF}
+    echo "Database is already initialzed"
+    return
+  fi
+
+  echo "Creating cluster '${PG_CLUSTER_NAME}' with data dir '${PGDATA}'"
+  mkdir -p "${PGDATA}" && chown -R postgres:postgres "${PGDATA}"
+  su postgres -c "pg_createcluster -d ${PGDATA} --start-conf auto ${PG_VERSION} ${PG_CLUSTER_NAME}"
+  # mv conf to $PGDATA and link it
+  cp -pr ${PG_CLUSTER_CONF}/* ${PGDATA} && \
```

Why are we using `&&` everywhere? `set -e` is in effect so we can safely use one command per line and it will fail fast.

---

updated